deploy/examples/ingress-with-annotations.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+spec:
+  strategy: allInOne 
+  storage:
+    type: memory 
+    options: 
```

Comment:
Could you please leave the example with only the minimum set of fields required to work?